index.js

```diff
@@ -1,13 +1,15 @@
 var marked = require('marked');
-// var output = require('lib/output');
 
-function parseMarkdown(text, debug) {
+function parseMarkdown(text, inline=false) {
   const latexMatcher = /(\$\$[\s\S][^$]+\$\$)/g;
   const latexPlaceholder = 'LATEXPLACEHOLDER';
   const matches = text.match(latexMatcher);
   const textWithoutLatex = text.replace(latexMatcher, latexPlaceholder);
 
-  var str = marked(textWithoutLatex);
+  if(inline) {
+      var str = marked.inlineLexer(textWithoutLatex, []);
+  } else
+      var str = marked(textWithoutLatex);
```

Comment:
Could you please add a space and the `if` and surround the `else` block with braces?

Reply:
Done!